src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -65,11 +91,17 @@ def build(self) -> SlackBlock:
             project_text = f"Here's what happened in the *{project.slug}* project today:"
             blocks.append(self.get_markdown_block(project_text))
 
+            fields = []
+            event_count_text = "*Today’s Event Count*: "
```

Comment:
I moved this up so if there isn't 14 days worth of past data to calculate the percent difference, we still show the event count for the day.